api/src/resources/Transaction/resolver.ts

```diff
@@ -37,104 +35,55 @@ export class ProductResolver {
   @Mutation((returns: void) => TransactionPayload)
   public async startMembershipTransaction(
     @Ctx() context: IContext,
-    @Arg("membershipType", (type: void) => MembershipTypes)
+    @Arg("membershipType", (retuns: void) => MembershipTypes)
```

Comment:
```suggestion
    @Arg("membershipType", (returns: void) => MembershipTypes)
```

Reply:
They won't technically as this is never run, but we should probably remove this function eventually